app/models/assignment_activity.rb

```diff
@@ -14,27 +15,31 @@ def self.from(audit, appointment)
   end
 
   def self.create_assignment(audit, appointment)
-    activity = create!(
+    create!(
       user_id: audit.user_id,
       message: ASSIGNED,
       appointment: appointment,
       owner: appointment.guider
     )
-
-    PusherActivityNotificationJob.perform_later(appointment.guider_id, activity.id)
   end
 
   def self.create_reassignment(audit, appointment)
     prior_owner = User.find(audit.audited_changes['guider_id'].first)
-    activity = create!(
+    create!(
       user_id: audit.user_id,
-      message: 'reassigned',
+      message: REASSIGNED,
       appointment: appointment,
       owner: appointment.guider,
       prior_owner_id: prior_owner.id
     )
+  end
 
-    PusherActivityNotificationJob.perform_later(appointment.guider_id, activity.id)
-    PusherActivityNotificationJob.perform_later(prior_owner.id, activity.id)
+  def pusher_notify_user_ids
+    case message
```

Comment:
For now this could just collapse to a ternary perhaps? `message == ASSIGNED ? owner_id : Array(owner_id, prior_owner_id)` - if you feel that reads better? I'm not _overly_ fussed though so feel free to ignore.